### PR TITLE
Improve dependencies

### PR DESCRIPTION
**Pull request by joneff**
The changes in a nutshell:

* cleanup package.json and generate lock files
* cleanup local .gitignore files and move declarations to root .gitignore
* use a single editorconfig file
* use a single codewoners file
* moved editorconfig-tools to root package.json

The cleanup of package files is needed to properly run the themes with nodejs v8+ on windows 10. There were some issues and the cleanup targets them.

The rest of the commits are nice to have.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
+.vscode/
+.tmp/
+node_modules/
+dist/
+modules/
+
+# Log files
+npm-debug.log
 lerna-debug.log
+debug.log
```
Comment (joneff) [inline .gitignore]: Not sure if we should have this entry

File: CODEOWNERS.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
 * @gyoshev @joneff
-docs/* @dmihaylo
+docs/ @dmihaylo
```
Comment (joneff) [inline CODEOWNERS.md]: As per https://help.github.com/articles/about-codeowners/, the modification will match all docs/ directories, as well as all their child directories.

File: packages/default/package.json [hunk 3]
```diff
@@ -21,7 +21,7 @@
     "lint": "sass-lint -v -c ./sass-lint.yml",
     "api": "sassdoc ./scss/",
     "api-check": "npm run api && git diff --exit-code --quiet -- docs/ || (echo -e '\\033[0;31mERROR: API docs are out of date' && exit 1)",
-    "start": "webpack-dev-server",
+    "start": "webpack-dev-server --hot --inline",
```
Comment (joneff) [inline packages/default/package.json]: tweaked the args to match the other themes

File: .gitignore (hunk 1 above)
Comment (gyoshev) [reply .gitignore]: Not sure either, but doesn't hurt.

**Pull request merged**